src/commands/helm-conftest.yaml

```diff
@@ -0,0 +1,25 @@
+---
+parameters:
+  chart:
+    type: "string"
+  policy_path:
+    description: Path to the Rego policy files directory.
+    type: string
+    default: policy
+description: Test conftest policies
+steps:
+  - run: |
+      if [ "<< parameters.policy_path >>" == "policy" ]; then
+          mkdir -p policy
+          conftest pull --policy policy https://raw.githubusercontent.com/swade1987/deprek8ion/6b1c53db2d48c9361c61d73dab3594f01324f68a/policies/1.16-deprek8ion.rego
+          conftest pull --policy policy https://raw.githubusercontent.com/swade1987/deprek8ion/6b1c53db2d48c9361c61d73dab3594f01324f68a/policies/1.17-deprek8ion.rego
+          conftest pull --policy policy https://raw.githubusercontent.com/swade1987/deprek8ion/6b1c53db2d48c9361c61d73dab3594f01324f68a/policies/1.18-deprek8ion.rego
+          conftest pull --policy policy https://raw.githubusercontent.com/swade1987/deprek8ion/6b1c53db2d48c9361c61d73dab3594f01324f68a/policies/1.19-deprek8ion.rego
+          conftest pull --policy policy https://raw.githubusercontent.com/swade1987/deprek8ion/6b1c53db2d48c9361c61d73dab3594f01324f68a/policies/1.20-deprek8ion.rego
+      fi
+      if [ -d "helm/<<parameters.chart>>/ci" ]; then
+        for t in $(ls helm/<<parameters.chart>>/ci/*.yaml); do helm template --values $t helm/<<parameters.chart>> | conftest test --policy << parameters.policy_path >> -; done
+      else
+        helm template helm/<<parameters.chart>> | conftest test --policy << parameters.policy_path >> -
+      fi
```

Comment:
Please check https://github.com/giantswarm/architect-orb/#design-and-goals

> - Having one line binary call on each job step - to make the outputs grouped
  and visible and keeping build configurations sane. Good example is
  [helm-chart-template][helm-chart-template].

Reply:
But I'd work out if we need the parameter in the first place.